### ISSUE TITLE
tip of tooltip always left-aligned since 3.2.7

### ISSUE THREAD

**Issue by phloxic** [flowplayer/flash#58]
Becomes obvious when you have a longer localization of "Fullscreen", the tooltip is in danger not to be associated with the correct button.

See: http://download.g63.ru/trash/tooltip_bug.png
Above: fp 3.2.5

Has the alignment just changed from right to left? Or was it more flexible previously? Should be tested with long text e.g. for Pause and Fullscreen.

See also: flash.flowplayer.org/forum/8/104475

**Comment by danrossi** [comment]
hi confirming its messed up its not taking some allign setting into account i think.

**Comment by danrossi** [comment]
https://dl.dropbox.com/u/3394987/flowplayer.controls-3.2.14.zip

**Comment by danrossi** [comment]
fixed closing for now.

**Comment by phloxic** [comment]
Well, now _all_ tooltips are right aligned. Try this:

``` js
controls: {
  stop: true,
  tooltips: {
    buttons: true,
    stop: "stop this movie"
  }
}
```

If we do not allow additional options - better not - the stop, play/pause, next, prev, speed? button tooltips must be left aligned.  Mute, volume, fullscreen right aligned. - Well, depending on which side of the player the buttons are.

**Comment by danrossi** [comment]
Good call you are right there is issues however it detecting this, which side of what ? It just sees if its clipping against the parent area of the stage and chooses which alignment from that. I'll see if they can be moved to the right also.

**Comment by danrossi** [comment]
ok sorry about that the bug is caused by being unable to obtain the position of the button in relative to the stage, the alignments are only for the most left and right button and everything else should be centre. If you like I can add an alignment config here also.

**Comment by danrossi** [comment]
https://dl.dropbox.com/u/3394987/flowplayer.controls-3.2.14.zip please try this. only if the tooltip will overflow the left and right areas of the stage should it align or else it will stay centred.

**Comment by danrossi** [comment]
Leaving this open until release.

**Comment by anssip** [comment]
Seems to be fixed now. I am closing all issues that i can verify to have been fixed.

**Comment by phloxic** [comment]
confirming that it's fixed in 3.2.17 dev.
